packages/backend/server/src/config/affine.env.ts

```diff
@@ -15,8 +15,13 @@ AFFiNE.ENV_MAP = {
   OAUTH_GOOGLE_CLIENT_SECRET: 'plugins.oauth.providers.google.clientSecret',
   OAUTH_GITHUB_CLIENT_ID: 'plugins.oauth.providers.github.clientId',
   OAUTH_GITHUB_CLIENT_SECRET: 'plugins.oauth.providers.github.clientSecret',
+  OAUTH_OIDC_ISSUER: 'plugins.oauth.providers.oidc.issuer',
   OAUTH_OIDC_CLIENT_ID: 'plugins.oauth.providers.oidc.clientId',
   OAUTH_OIDC_CLIENT_SECRET: 'plugins.oauth.providers.oidc.clientSecret',
+  OAUTH_OIDC_SCOPE: 'plugins.oauth.providers.oidc.args.scope',
```

Comment:
in what scenarios do we need to define different scope fields?
In affine login, we only need to get the username, email address and id as the unique identifier of the account

Reply:
the default value, `openid profile email`, is OK. but I set this for the future, maybe we want more scope, e.g. `groups`.